Restore block movers to focus mode

kjellr opened the pull request:
Closes #14670. 

Currently Focus mode does not include a way to rearrange blocks. This PR corrects that by re-adding block movers to that mode. Conceptually, these seem fine to re-introduce because the movers shouldn't necessarily break your focus anyway: they only appear when you hover on a specific part of a block, and they fade away when you type. 

**Before:** 

<img width="484" alt="Screen Shot 2019-04-22 at 12 26 40 PM" src="https://user-images.githubusercontent.com/1202812/56511158-e568dd80-64f9-11e9-83e5-93059e3fca3b.png">


**After:**

<img width="443" alt="Screen Shot 2019-04-22 at 12 26 32 PM" src="https://user-images.githubusercontent.com/1202812/56511162-e7cb3780-64f9-11e9-82f4-7c7157cb38ce.png">

kjellr committed:
Restore block movers to focus mode.

draganescu commented:
hi @kjellr I ran the code and the block mover is too close to the block toolbar, tried with twentynineteen and twentyfourteen

<img width="804" alt="Screenshot 2019-04-22 at 20 05 11" src="https://user-images.githubusercontent.com/107534/56513176-46b0a100-653a-11e9-9a60-e7810dbb9c20.png">
<img width="556" alt="Screenshot 2019-04-22 at 20 04 23" src="https://user-images.githubusercontent.com/107534/56513177-46b0a100-653a-11e9-8108-a554aa432f6e.png">
<img width="603" alt="Screenshot 2019-04-22 at 20 03 58" src="https://user-images.githubusercontent.com/107534/56513178-4a442800-653a-11e9-827b-436473c200f8.png">

kjellr committed:
Remove unnecessary negattive margins on the block toolbar.

These were only included in focus mode, and end up causing overlap with the block mover butons.

kjellr commented:
Good catch, @draganescu! It's weird that didn't show up for me initially. 🤔 

Anyway, I removed the style that was causing that in ecc932f.

jasmussen commented:
Yep, works for me 👍 👍

Just needs a code sanity check, I suppose!

kjellr commented:
Thanks, everyone!
